aether-kernel/entrypoint.sh

```diff
@@ -116,7 +116,12 @@ function setup {
 
     STATIC_ROOT=${STATIC_ROOT:-/var/www/static}
     # create static assets
-    ./manage.py collectstatic --noinput --clear --verbosity 0
+    echo "Collecting static files..."
+    if [ $COLLECT_STATIC_FILES_ON_STORAGE ]; then
```

Comment:
take in mind that the file has the `-u` option enabled, if `COLLECT_STATIC_FILES_ON_STORAGE` does not exist, the command throws an error.

> **-u**
> Treat unset variables and parameters other than the special parameters ‘@’ or ‘*’ as an error when performing parameter expansion. An error message will be written to the standard error, and a non-interactive shell will exit.

Reply:
you could leave the `./manage.py collectstatic --noinput --verbosity 0` for both cases :woman_shrugging:

Reply:
Curiosity, why couldn't we clear the folder content while collecting the static content?

Reply:
ACL permissions.